app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.wife = None
+        self.husband = None
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    new_list = [Person(p["name"], p["age"]) for p in people]
+    for person_id in range(len(new_list)):
+        his_wife = people[person_id].get("wife")
+        if his_wife is not None:
+            for wife_id in new_list:
+                if wife_id.name == his_wife:
+                    new_list[person_id].wife = wife_id
```

Comment:
Można to troszkę uprościć. 
```suggestion
        person_instance = Person.people[person["name"]]
        if "wife" in person and person["wife"]:
            person_instance.wife = Person.people[person["wife"]]
```
i tak naprawdę możemy dodać tutaj podobny warunek, tylko że dla `husband`